src/core/blob.cpp

```diff
@@ -20,31 +26,37 @@ blob::blob(session & s)
 
 blob::~blob()
 {
-    delete backEnd_;
+    if (backEnd_)
```

Comment:
Good to know :-)  I was believing `delete` works like `free` and you can not call `free` on NULL pointer.